src/lib/components/common/Checkbox.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+  import { createEventDispatcher } from "svelte";
+  import { Check16, Dash16 } from "svelte-octicons";
+
+  const dispatch = createEventDispatcher();
+
+  export let disabled = false;
+  export let status: "off" | "on" | "some" = "off";
```

Comment:
I think it'd be better to have two booleans here, like in both `Button` above and the [HTML `input` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes): `checked` and `indeterminate`.